### PR TITLE
docs(component): update readme for serving static files

### LINKED ISSUE #51: Throws not found for static files
**Describe the bug**
The action provided by this package throws a not found error for static files

**To Reproduce**
1. Add the authorize action to the sequence
2. Get any static file
3. It throws not found even when the file or endpoint is in ignore list

**Expected behavior**
The endpoint should be gettable if in ignore list

### PR DESCRIPTION
**Pull request by Surbhi-sharma1**
update readme for serving static files

GH-51

## Description
Updated readme for serving static files.

Fixes #51 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -267,6 +267,15 @@ export const enum PermissionKey {
...
 ```
+- Serving the static files:
+
+Authorization  configuration binding sets up paths that can be accessed without any authorization checks, allowing static files to be served directly.The allowAlwaysPaths property is used to define these paths for the files in public directory  i.e for a test.html file in public directory ,one can provide its path as follows:
+
+```
+this.bind(AuthorizationBindings.CONFIG).to({
+  allowAlwaysPaths: ['/explorer',''/test.html'],
+});
+```
```
Comment (yeshamavani) [inline README.md:278]: also mention that the public folder should be added to static 
and if the file is in some other folder then.. one more app.static has to be added.

**Review by yeshamavani — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Surbhi-sharma1) [reply README.md:278]: ok

Comment (Surbhi-sharma1) [reply README.md:278]: Done.

**Pull request merged**